packages/apps/admin/src/components/Admin/StyledComponents.js

```diff
@@ -2,23 +2,22 @@ import SlideMenu from 'react-burger-menu/lib/menus/slide'
 import styled from 'styled-components'
 import {theme} from 'tocco-ui'
 
-const calcViewportHeight = () => {
-  const vh = window.innerHeight * 0.01
-  document.documentElement.style.setProperty('--vh', `${vh}px`)
+const documentHeight = () => {
+  const doc = document.documentElement
+  const viewportHeight = Math.max(document.documentElement.clientHeight || 0, window.innerHeight || 0)
+  doc.style.setProperty('--doc-height', `${viewportHeight}px`)
 }
 
-calcViewportHeight()
-
-window.addEventListener('resize', () => calcViewportHeight())
-window.addEventListener('orientationchange', () => calcViewportHeight())
+documentHeight()
+window.addEventListener('resize', () => documentHeight())
```

Comment:
suggestion (blocking): Move height calculation into a separate component.

I know it was already like that before... but.. when having this logic in a react component we would have the `addEventListener` under control and we wouldn't need to add a css variable.

Does this then still work?

```
const Wrapper = ({children}) => {
  const docHeight = useRef(null)

  const documentHeight = () => {
    const viewportHeight = Math.max(document.documentElement.clientHeight || 0, window.innerHeight || 0)
    docHeight.current = `${viewportHeight}px`
  }

  useEffect(() => {
    documentHeight()
    window.addEventListener('resize', documentHeight)

    return () => {
      window.removeEventListener('resize', documentHeight)
    }
  }, [])

  return <StyledWrapper height={docHeight.current}>{children}</StyledWrapper>
}
```